extension/persistence/relational-jdbc/src/main/resources/postgres/schema-v1.sql

```diff
@@ -15,16 +15,12 @@
 -- KIND, either express or implied.  See the License for the
 -- specific language governing permissions and limitations
 -- under the License.
---
 
--- Note: Database and schema creation is not included in this script. Please create the database and
--- schema before running this script. for example in psql:
--- CREATE DATABASE polaris_db;
--- \c polaris_db
--- CREATE SCHEMA polaris_schema;
--- set search_path to polaris_schema;
+CREATE SCHEMA IF NOT EXISTS POLARIS_SCHEMA;
+SET search_path TO POLARIS_SCHEMA;
 
 CREATE TABLE IF NOT EXISTS entities (
+    realm_id TEXT NOT NULL,
```

Comment:
Minor concern: the length TEXT is unbounded. It performs not well in case of a long string, esp. it's a PK, could be part of join. Should we limit its length? We don't have to do at the schema level though.

Reply:
Yes, i think its a good idea, generally if we are imagining it to be an account-id type thing 20 should be fine, worst case we can make this 50 incase people wanna encode more info / meaning to it.
@dimas-b thoughts ?

Reply:
Nessie uses `VARCHAR` for "repository" IDs. I do not think it's substantially different from `TEXT`.

Does max field size matter in Postgres if actual values are small?

Ultimately the size of realmID is a Polaris Administrator's concern. I would not want to put arbitrary limits on it, if it does not affect Polaris code behaviour.

A bigger concern, IMHO, is case sensitivity. I believe we should ensure that realm IDs are case sensitive in the database.

Reply:
did some deep-dive per PGSQL : https://wiki.postgresql.org/wiki/Don't_Do_This#Text_storage  shouldn't and this is prefered over char(n), varchar(n) 

for the case-sensitivity are suggesting marking column something as 

```
CREATE TABLE my_table (
    id SERIAL PRIMARY KEY,
    sensitive_data TEXT COLLATE "C"
);
```
can this be configured as default at database level ? seems like the default of Postgres is case insensitive.

Reply:
Let's do collation settings in a follow-up PR, but we must not forget it :)